cert/selfsigned_test.go

```diff
@@ -185,3 +224,63 @@ func TestTLSDisableAutofill(t *testing.T) {
 		"TLS Certificate was not marked as outdated when it should be",
 	)
 }
+
+// TestTlsConfig tests to ensure we can generate a TLS Config from
+// a tls cert and tls key.
+func TestTlsConfig(t *testing.T) {
+	tempDir := t.TempDir()
+	certPath := tempDir + "/tls.cert"
```

Comment:
Should use `filepath.Join()` to make this work reliably on all operating systems.